### PR TITLE
add docker info for JavaCLI testing

### PR DESCRIPTION
**Pull request by DenverM80**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM greyrocksoftware/javastandalone
```
Comment (rpmoore) [inline docker/Dockerfile]: This should use `javastandalone7`

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/test/helpers/ABMTestHelper.java [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+            fail("Data policy already exists, terminating to prevent conflict: " + dataPolicyName);
+        } catch (final IOException e) {
+            //Pass: expected data policy to not exist
+        }
+
+        if (checksumType == null) {
+            //Create the data policy with versioning
+            return client.putDataPolicySpectraS3(new PutDataPolicySpectraS3Request(dataPolicyName)
+                    .withVersioning(versioningLevel));
+                    //.withAlwaysForcePutJobCreation(true));
```
Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/test/helpers/ABMTestHelper.java]: This should be removed unless it is going to be added back later.  If it is going to be added back, add a TODO

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/test/helpers/ABMTestHelper.java [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+        }
+        //Create the data policy with versioning and checksum
+        return client.putDataPolicySpectraS3(new PutDataPolicySpectraS3Request(dataPolicyName)
+                .withVersioning(versioningLevel)
+                .withEndToEndCrcRequired(true)
+                .withChecksumType(checksumType));
+                //.withAlwaysForcePutJobCreation(true));
```
Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/test/helpers/ABMTestHelper.java]: This should be removed unless it is going to be added back later. If it is going to be added back, add a TODO

Comment (DenverM80) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/test/helpers/ABMTestHelper.java]: I think its only available with 3.2+

We are shooting for 3.0.x compatibility at this point, right?  If so I'll go ahead and remove - just working through some issues to get all of the tests passing on a freshly PXE'd BP

Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/test/helpers/ABMTestHelper.java]: If that is the case, leave a TODO explaining why the code is commented out.

**Pull request merged**